counterpartylib/lib/transaction.py

```diff
@@ -158,7 +158,12 @@ def construct_coin_selection(encoding, data_array, source, allow_unconfirmed_inp
                 filtered_unspent.append(output)
         unspent = filtered_unspent
 
-        unspent = backend.sort_unspent_txouts(unspent)
+        if encoding == 'multisig':
+            dust = config.DEFAULT_MULTISIG_DUST_SIZE
+        else:
+            dust = config.DEFAULT_REGULAR_DUST_SIZE     
+            
+        unspent = backend.sort_unspent_txouts(unspent, dust_size=dust)
```

Comment:
I don't think we need to change utxo filtering based on if tx is multisig or not... what we care about is getting spendable utxos to use as inputs, and using DEFAULT_REGULAR_DUST_SIZE (546) will return all usable utxos to use as inputs.

I know for multisig the outputs need to be at least `DEFAULT_MULTISIG_DUST_SIZE` (7800), but Is there a reason we only want to use inputs greater than 7800 with multisig?